R/docs_extra.R

```diff
@@ -25,7 +25,7 @@
 #' associated with Chi-squared and F distributions), this applies also to cases
 #' where the lower bound of the CI is equal to 0. Even more care should be taken
 #' when the *upper* bound is equal to 0 - this occurs when *p*-value is greater
-#' than 1−alpha/2 making, the upper bound unestimatable, and the upper bound is
```

Comment:
@IndrajeetPatil This was actually correct... 
Maybe "cannot be estimated" instead?

Reply:
Done!